pkg/infradb/taskmanager/taskmanager.go

```diff
@@ -123,7 +130,18 @@ func (t *TaskManager) processTasks() {
 				// (e.g. Maybe you have a timeout on the subscribers and you got the notification after the timeout have passed)
 				NotificationID: uuid.NewString(),
 			}
-			eventbus.EBus.Publish(objectData, sub)
+			if err := eventbus.EBus.Publish(objectData, sub); err != nil {
+				log.Printf("processTasks(): Notification not sent to subscriber %+v with data %+v. Subscriber is busy. The Task %+v will be requeued.\n", sub, objectData, task)
+				// We keep this subIndex in order to know from which subscriber to start iterating after the requeue of the Task
+				// so we do start again from the subscriber that returned an error or was unavailable for any reason.
+				task.subIndex += i
+				task.systemTimer *= 2
```

Comment:
- I can change the name of the var that is no problem.
- The timer will increase every time that we try to publish but the channel is busy and we need to reque the task. So it is working this way:

1. We publish
2. An error is returned because the channel is busy
3. We increase the timer
4. We wait for the timer to expire in a goroutine and then we requeue the task

Reply:
The flow you described is clear. I want to know if we ever stop increasing it. Will it make to wait for hours? days? years?

Reply:
Currently we never stop increasing it. But this is a known issue that we have just not addressed do far. We are planning to address this in the future. I can open an issue to track it.

Reply:
Pls create an issue

Reply:
https://github.com/opiproject/opi-evpn-bridge/issues/394